packages/pigeon/platform_tests/alternate_language_test_plugin/example/android/build.gradle

```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```

Comment:
?

Reply:
🤷 This is the diff the generator script produced when re-generating all the generated files.

Reply:
This looks like it was probably a `flutter` migrator from running on `master`. It's fine if you want to leave it, but it's unrelated to anything in this PR.

Reply:
I'll revert it.